atlasdb-client/src/main/java/com/palantir/atlasdb/schema/KvsRangeMigrator.java

```diff
@@ -78,6 +80,32 @@
         this.rowTransform = rowTransform;
     }
 
+    // Report (a) completion or (b) start row. Intended to be called before migration has started.
+    @Override
+    public void checkStatus(int size) {
+        txManager.runTaskWithRetry((TransactionTask<Void, RuntimeException>) transaction -> {
+            checkStatus(transaction, size);
+            return null;
+        });
+    }
+
+    private void checkStatus(Transaction tx, int numRangeBoundaries) {
+        for (int rangeId = 0; rangeId < numRangeBoundaries; rangeId++) {
+            byte[] checkpoint = getCheckpoint(rangeId, tx);
+            if (checkpoint != null) {
+                log.info("({}/{}) Migration from table {} to table {} will start/resume at {}",
+                        rangeId,
+                        numRangeBoundaries,
+                        srcTable,
+                        destTable,
+                        PtBytes.encodeHexString(checkpoint)
```

Comment:
This is the row key from which the migration will resume, as stored in the checkpoints table.

From the [javadoc](https://github.com/palantir/atlasdb/blob/0.53.0/atlasdb-client/src/main/java/com/palantir/atlasdb/schema/AbstractTaskCheckpointer.java#L48): " Get the next row name for this range. Returns null if the checkpoint is done."